src/test/java/net/snowflake/client/jdbc/ResultSetMultiTimeZoneLatestIT.java

```diff
@@ -23,7 +23,7 @@
   @Parameterized.Parameters(name = "format={0}, tz={1}")
   public static Collection<Object[]> data() {
     // all tests in this class need to run for both query result formats json and arrow
-    String[] timeZones = new String[] {"UTC", "Asia/Singapore", "MEZ"};
+    String[] timeZones = new String[] {"UTC", "Asia/Singapore", "MEZ", "Europe/London"};
```

Comment:
mega nit: It might be better.If we can add a comment here as well about the daylight factor in "Europe/London"

Reply:
sure, added comment